src/components/Firestore/Collection.module.scss

```diff
@@ -0,0 +1,16 @@
+.query-panel {
```

Comment:
Since we should move all to modules.. remove `.module.scss`, just `.scss`

Unless there's some specific reason it needs to be called that?